content/docs/render-props.md

```diff
@@ -305,14 +305,8 @@ To get around this problem, you can sometimes define the prop as an instance met
 
 ```js
 class MouseTracker extends React.Component {
-  constructor(props) {
-    super(props);
-
-    // This binding ensures that `this.renderTheCat` always refers
-    // to the *same* function when we use it in render.
-    this.renderTheCat = this.renderTheCat.bind(this);
-  }
-
+  // Defined as an instance method, `this.renderTheCat` always
+  // refers to *same* function when we use it in render
   renderTheCat(mouse) {
     return <Cat mouse={mouse} />;
   }
```

Comment:
In cases like this, I'd actually prefer to put it as a function completely outside the class to begin with. It's better for type checking (no `this` allowed), minified code size (local identifier, no associated object prop) and runtime memory size (don't need to keep a reference to the function in the prototype).